Weapons-Guide.md

```diff
@@ -9,17 +9,17 @@ In most games all players are equipped with a pistol and two of their favorite l
 
 ### Loadout weapons
 
-These weapons can be chosen in the *profile* game menu, and it is possible to change the weapon selection mid-match by pressing the *comma* key (per default). Note that one or both of the loadout weapon slots can be randomized.
+These weapons can be chosen in the *profile setup* game menu under *loadouts*, and it is possible to change the weapon selection mid-match by pausing and going through the menues. Note that one or both of the loadout weapon slots can be randomized, and the randomization option can be selected, although not per slot.
```

Comment:
`menus` does not have an `e`

Reply:
I am very confused here menu has an e in it
![image](https://user-images.githubusercontent.com/61996219/215324487-e632bfc4-bbbc-458b-a07c-71b08b13b543.png)